pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceConsumer.java

```diff
@@ -200,6 +203,9 @@ public void received(Consumer consumer, Message msg) {
                     limiter.acquire();
                 }
 
+                long latencyMicros = NANOSECONDS.toMicros(System.nanoTime() - msg.getPublishTime());
```

Comment:
Publish time is in "millis". That is because `nanoTime()` is not an absolute timing source and cannot be used across different machines.